src/commons/interfaces.ts

```diff
@@ -9,6 +9,7 @@ export interface InputPropsTypes {
   checked?: boolean
   onChange?: (event: boolean) => void // This has to be fixed
   reset?: boolean
+  invert?: string
```

Comment:
I think it's really not clear that invert is a string, it seems more a boolean to me

Reply:
Good point, I rename it to something that hopefully suggest the idea that we are expecting a string